web/pages/program/[slug].tsx

```diff
@@ -148,11 +148,32 @@ const SessionRoute = ({
     currentSessionInProgram?._id,
     sessions
   );
+  const hasAssociatedProgram = startDateTime && currentSessionInProgram;
```

Comment:
Nit: Grouping this with the other time-related variables, e.g. moving this line below the `hasHighlightedSpeakers` definition, might be a bit easier to read (less "context switching").

Reply:
Fixed as part of cfc3720e434795cb60c659003f8006d931d5ff4c.